web/ui/dashboard/src/app/private/components/create-project-component/create-project-component.component.ts

```diff
@@ -277,7 +283,7 @@ export class CreateProjectComponent implements OnInit {
 	}
 
 	confirmAction() {
-		this.projectForm.get('config.disable_endpoint')?.valueChanges.subscribe(selectedValue => {
+		this.projectForm.get('config.disable_endpoint')?.valueChanges.pipe(takeUntil(this.destroy$)).subscribe(selectedValue => {
```

Comment:
I think this is an overkill, why not do something like this:
```
disableEndpointInput = this.projectForm.get('config.disable_endpoint')?.valueChanges;

ngOnDestroy() {
    disableEndpointInput.unsubscribe();
}

confirmAction() {
		disableEndpointInput.subscribe(selectedValue => {
		...
```

Except it doesn't or can't work this way. @Oluwadaminiola

Reply:
This doesn't work, I also tried this:
`disableEndpointSubscription: Subscription;

ngOnDestroy(){
   disableEndpointSubscription.unsubscribe()
}

confirmAction() {
       this.disableEndpointSubscription = this.projectForm.get('config.disable_endpoint')?.valueChanges.subscribe
...
`

but it keeps throwing errors

@horlah